src/components/DigitalAssetsManage/DigitalAssetsManage.js

```diff
@@ -1,42 +1,52 @@
 // @flow
 
 import React, { PureComponent } from 'react'
-import { handle } from 'utils/eventHandlers'
+
+import handle from 'utils/eventHandlers/handle'
```

Comment:
These "handle" thing looks old enough, so I don't think it blocks PR from merging. But it would be much better if it was called `ignoreEventHandle` or something else more detailed than now.

Reply:
we already have `ignoreEvent` util method that calls preventDefault and then stopPropagation for event (to prevent bubbling). But we can rename it to `preventEvent`, and this method `handle` to `ignoreEvent`.

Actually I don't sure that its name should be so detailed, because it says `handle` ( `that function` ) ( `with these args` )